pyanaconda/payload/dnf/payload.py

```diff
@@ -405,6 +410,11 @@ def _set_up_additional_repository(self, data):
             )
             result = task.run()
             data.url = "file://" + result.install_tree_path
+            return
+
+        # Otherwise, raise an error.
+        msg = _("The '{}' repository has an unsupported protocol.")
+        raise SourceSetupError(msg.format(data.name)) from None
```

Comment:
The hint for translators would go also here (as above).

Reply:
Fixed.